src/mmcesupport.c

```
@@ -381,20 +381,24 @@ void mmceLoadModules(void)
     sysLoadModuleBuffer(&mmceman_irx, size_mmceman_irx, 0, NULL);
 }
 
+static int sGameIdTransportArmed = 0;
+
 // Δ4 (NHDDL parity): arm the GameID transport OUTSIDE the launch path. NHDDL loads mmceman once at
 // boot; RiptOPL used to self-arm inside mmceSendGameID -- an IRX load/start at the launch's most
 // fragile moment (issue #51's fix, right intent, wrong timing). Called from initAllSupport (boot +
 // every settings apply) via the IO worker, so a wedged load is a harmless LOG at menu time instead
 // of a dead launch. Idempotent (mmceLoadModules latches); no-op when the GameID feature is off.
 void mmceArmGameIDTransport(void)
```

Cppcheck (2.21.0)

[style] The function 'mmceArmGameIDTransport' is never used.

(unusedFunction)